File: wizer-angular-ui/src/app/models/BookSearchModel.ts
export class BookSearchModel {
  pageNo: number;
  pageSize: number;
  title: string;
  categoryName: string;
  author: string;
  isbn: string;

  constructor() {
    this.pageNo = 0;
    this.pageSize = 10;
    this.title = '';
    this.categoryName = '';
    this.author = '';
    this.isbn = '';
  }

}

File: wizer-service/src/main/java/com/wizer/test/wizer/assessment/dto/request/BookSearchModel.java
package com.wizer.test.wizer.assessment.dto.request;

import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter @Setter @NoArgsConstructor
public class BookSearchModel {

    private String author;

    private String title;

    private String isbn;

    private String categoryName;

    private Integer pageNo = 0;

    private Integer pageSize = 10;

}
